skbio/metadata/tests/test_mixin.py

```diff
@@ -0,0 +1,77 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+
+import unittest
+
+from skbio._base import (MetadataMixin, PositionalMetadataMixin)
+from skbio.util._decorator import overrides
+from skbio.util._testing import (ReallyEqualMixin, MetadataMixinTests,
```

Comment:
Sorry I didn't notice this before: I think `MetadataMixinTests` and `PositionalMetadataMixinTests` should be moved to `skbio.metadata._testing` since they are mixin tests for testing metadata mixins